schedule/yast/autoyast/autoyast_pxe_boot.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: autoyast_pxe_boot
```

Comment:
I found in another test suite that with this generalization on the name of the test suite we lost the info from the summary about this test suite and once we add just one validation to this scenario we need to rename everything and fill it again. We miss validation everywhere, so just please consider if it worth it to follow this pattern.

Reply:
Validation is out of scope here. @JRivrain is already working on adding validation to `autoyast_lvm` test (https://progress.opensuse.org/issues/68398), so we will get different schedules. What we could do though is to use include tag and move common part to separate file.

Reply:
> What we could do though is to use include tag and move common part to separate file.

@rwx788 As far as I remember it was not possible to use `!include` with array and then merge two arrays. Has it already implemented?

Reply:
It would be great to use the include feature with a list, I'm not sure if it is possible or how it is the syntax, @OleksandrOrlov told me yesterday that wasn't, only with a yaml key, maybe we are missing something, using some anchor or other trick.

Reply:
Yes, it won't be easy. I think we will have to use fake node for the import, assign anchor to it and then do merge, if the list merge doesn't work. But I need to try it myself first.